.github/workflows/quarto-publish.yml

```
@@ -82,6 +82,8 @@ jobs:
           to: html
 
       - name: JamPack Optimization
+        # FIXME: JamPack does not finish on ubuntu-latest for unknown reasons
+        if: github.repository =! 'maehr/open-research-data-template'
```

actionlint (1.7.8)

got unexpected character '!' while lexing == operator, expecting '='

(expression)